components/gitpod-db/go/oidc_client_config.go

```diff
@@ -278,6 +285,19 @@ func setClientConfigActiveFlag(ctx context.Context, conn *gorm.DB, id uuid.UUID,
 	if tx.Error != nil {
 		return fmt.Errorf("failed to set oidc client config as active to %d (id: %s): %v", value, id.String(), tx.Error)
 	}
+
+	if active {
+		tx := conn.
+			WithContext(ctx).
+			Table((&OIDCClientConfig{}).TableName()).
+			Where("id != ?", id.String()).
+			Where("organizationId != ?", config.OrganizationID).
```

Comment:
Just getting a workspace to check if the table also has the `deleted column` which we may need to check here